contracts/interfaces/IRelayHub.sol

```diff
@@ -114,19 +112,19 @@ interface IRelayHub {
 
 
     /// Relays a transaction. For this to succeed, multiple conditions must be met:
-    ///  - canRelay must return CanRelayStatus.OK
-    ///  - the sender must be a registered relayWorker
-    ///  - the transaction's gas price must be larger or equal to the one that was requested by the sender
+    ///  - Paymaster's "acceptRelayCall" method must succeed and not revert
+    ///  - the sender must be a registered Relay Worker that the user signed
+    ///  - the transaction's gas price must be equal or larger than the one that was signed by the sender
     ///  - the transaction must have enough gas to run all internal transactions if they use all gas available to them
-    ///  - the sponsor must have enough balance to pay the relayWorker for the scenario when all gas is spent
+    ///  - the Paymaster must have enough balance to pay the Relay Worker for the scenario when all gas is spent
     ///
     /// If all conditions are met, the call will be relayed and the recipient charged.
     ///
     /// Arguments:
-    /// @param relayRequest - all details of the requested relayWorker call
-    /// @param signature - client's signature over all previous params, plus the relayWorker and RelayHub addresses
+    /// @param relayRequest - all details of the requested relayed call
+    /// @param signature - client's EIP-712 signature over the relayRequest struct
     /// @param approvalData: dapp-specific data forwarded to acceptRelayedCall.
```

Comment:
nitpick: its Paymaster-specific data, not dapp-specific

Reply:
this string was not even touched by this commit

Reply:
exactly... you changed all the documents around it - but it not (still, its a nitpick only: I'd accept it if that was the only issue: this commit is based on another commit to be fixed